### DISCUSSION TITLE
Deployment

### DISCUSSION THREAD

**Discussion by paultit** [vercel/vercel#4526] [Help] [answered]
Hello, help me how can I change the settings for the deployment project, can't found where to do it ?

**Comment by coetry** [comment, marked as answer]
Hi @paultit if you click on your project, on the top menu bar, you should see a link to _Settings_.

Additionally, it would follow this url pattern: `vercel.com/<username>/<project-name>/settings`.

**Comment by paultit** [comment]
Thank you, I found.

пт, 29 мая 2020 г. в 17:29, Allen Hai <notifications@github.com>:
